packages/react-button/src/components/CompoundButton/useCompoundButtonStyles.ts

```diff
@@ -157,8 +157,7 @@ const useContentContainerStyles = makeStyles({
 const useSecondaryContentStyles = makeStyles({
   base: theme => ({
     lineHeight: '100%',
-    marginTop: buttonSpacing.smaller,
-
+    marginTop: '4px',
```

Comment:
Why is this change being made?

Reply:
It was to resolve the error below when running `yarn storybook` in the `react-button` folder.

![image](https://user-images.githubusercontent.com/8649804/122551586-0948e080-cfea-11eb-9608-d23dd74e3dd9.png)